src/Consumer/CleanupSnapshotConsumer.php

```diff
@@ -41,6 +47,15 @@ class CleanupSnapshotConsumer implements ConsumerInterface
      */
     public function __construct(SnapshotManagerInterface $snapshotManager, PageManagerInterface $pageManager)
     {
+        @trigger_error(
```

Comment:
@VincentLanglet should I add this deprecate message here or just the [`@deprecated`](https://github.com/sonata-project/SonataPageBundle/pull/1434/files#diff-572a33f9239c5fe1ab83911ea8986a3a6d7961771d8edc182a70c2efea8d3f6eR30) on top of class is enough?

Reply:
It's better with the deprecation in the constructor like you did i would say.

Reply:
I thought the same, but it will broken the all tests that are not legacy

What do you think I add the `legacy group` for test that use this baseCommand and in the next major remove the group?